okgo/check/check.go

```diff
@@ -49,40 +112,26 @@ func Run(projectParam okgo.ProjectParam, checkersToRun []okgo.CheckerType, pkgPa
 
 	// sort the checkers
 	if err := sortCheckers(checkers); err != nil {
-		return err
-	}
-
-	jobs := make(chan okgo.CheckerParam)
-	results := make(chan checkResult, len(checkers))
-
-	// if there are fewer checkers than max parallelism, update parallelism to number of checkers
-	if len(checkers) < parallelism {
-		parallelism = len(checkers)
-	}
-
-	for w := 0; w < parallelism; w++ {
-		go singleCheckWorker(pkgPaths, projectDir, maxTypeLen, parallelism > 1, jobs, results, stdout)
+		return nil, 0, err
 	}
+	return checkers, maxTypeLen, nil
+}
 
+func partitionCheckerJobs(checkers []okgo.CheckerParam) ([]okgo.CheckerParam, []okgo.CheckerParam, error) {
+	var checkersWeCannotRunInParallel []okgo.CheckerParam
+	var checkersWeCanRunInParallel []okgo.CheckerParam
```

Comment:
Minor semantic preferences:
* Avoid use of "we"
* For this function, model bucketing as "isMultiCPU" vs. "not" -- level above is where we interpret this as being able to be run in parallel or not

```suggestion
var (
	multiCPUCheckers []okgo.CheckerParam
	singleCPUCheckers []okgo.CheckerParam
)
```

Reply:
Done